tests/security/ima/ima_appraisal_hashes.pm

```diff
@@ -27,7 +27,10 @@ sub run {
 
     add_grub_cmdline_settings("ima_appraise=fix $tcb_cmdline", update_grub => 1);
 
-    power_action('reboot', textmode => 1);
+    record_info("bsc#1189988: ", "We need disable secureboot with ima fix mode");
```

Comment:
Looks like it can use ' ' too

`('bsc#1189988: ', 'We need disable secureboot with ima fix mode');`